frontend/src/scenes/insights/Histogram/Histogram.tsx

```diff
@@ -206,51 +206,49 @@ export function Histogram({
                 _svg.node().appendChild(_bars.node())
 
                 // text labels
-                if (!isDashboardItem) {
```

Comment:
@alexkim205 was there a particular reason we don't show these labels on dashboard item funnel histograms?

Reply:
Hm, I can't remember but I don't see why we shouldn't anymore.